### ISSUE TITLE
vscode.previewHtml is deprecated. Need to be replaced with Webview API

### ISSUE THREAD

**Issue by muojp** [atsushieno/vscode-language-review#23]
VSCode warns us that vsccode-language-review uses deprecated preview API.

VSCode: 1.32.2
![image](https://user-images.githubusercontent.com/766864/54665598-d4a7ff00-4b2a-11e9-8ee9-4934164bbbe8.png)

**Comment by muojp** [comment]
https://github.com/Microsoft/vscode/issues/62630

**Comment by muojp** [comment]
A memo after reading https://code.visualstudio.com/api/extension-guides/webview .

A quick review of current design:

1. Create a preview panel using vscode.commands.executeCommand('vscode.previewHtml', ...) when user needs a new panel, then VSCode core is responsible for lifecycle of the panel.
2. Bind preview panels with Re:VIEW documents using URIs
3. Utilize vscode.TextDocumentContentProvider to 1/ detect changes, 2/ fire `convert` function, 3/ update content shown in preview panels (content update is done internally by VSCode using URI handling)

What Webview API requires:

1. Create a preview panel using vscode.window.createWebviewPanel(...)
2. Programmer is responsible for everything

In short, we would need to adopt Webview like this way:

 - Throw away TextDocumentContentProvider (`review:` URI scheme handler)
 - Re-bind editor events (onOpen/onChange/onSave) with corresponding preview panels along with `vscode.SymbolInformation`s

**Comment by atsushieno** [comment]
Wow, thanks for the note. I'll take a look hopefully tonight or in a few days.

**Comment by atsushieno** [comment]
okay, after a quick look, this is going to be annoying :p

**Comment by atsushieno** [comment]
Fixed. Thanks for the report!
